### PR TITLE
Bump version to 1.7.2

### PR DESCRIPTION
**Pull request by zhonghui12**
*Issue #, if available:*

*Description of changes:*
Bump version to 1.7.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.7.2
+* Enhancement - Fix aggregator size estimation (#155)
+* Enhancement - Fix partition key computation for aggregation (#158)
```
Comment (PettitWesley) [inline CHANGELOG.md]: are these actually enhancements or are they bugs that were fixed?

Comment (hossain-rayhan) [reply CHANGELOG.md]: I would say bug fix.

Comment (zhonghui12) [reply CHANGELOG.md]: I checked in the PR details and they said there should be some bugs. I will update the PR to make it more accurate

Comment (zhonghui12) [reply CHANGELOG.md]: Update them as bugs

**Review by hossain-rayhan — APPROVED** [review]

**Pull request merged**